draft-ietf-rtcweb-jsep.xml

```diff
@@ -877,6 +877,45 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                 </section>
             </section>
+
+            <section  title="Simulcast" anchor="sec.simulcast">
+
+                <t>JSEP supports simulcast of a MediaStreamTrack, where
+                multiple encodings of the source media can be transmitted within
+                the context of a single m= section. Receivers that understand
+                simulcast and are willing to receive it will include SDP markings
+                to indicate this support, and JSEP endpoints will use these
+                markings to determine whether simulcast is permitted for a given
+                RTPSender. If simulcast support is not negotiated, the RTPSender
+                will only be permitted to use a single encoding.</t>
```

Comment:
Perhaps:
"JSEP supports simulcast of a MediaStreamTrack, where
multiple encodings of the source media can be transmitted within
the context of a single m= section. The current JSEP API is designed
to allow applications to send simulcasted media but only to receive
a single encoding. This allows for applications where browser clients
send multiple encodings to a conferencing server which then selects
a single encoding to send to each client.

Applications request support for simulcast by configuring multiple
encodings on an RTPSender, which causes the m= section to be
marked as containing multiple encodings, as described below. Receivers that understand
simulcast and are willing to receive it will include SDP markings
to indicate this support, and JSEP endpoints will use these
markings to determine whether simulcast is permitted for a given
RTPSender. If the JSEP endpoint is the answerer, then the RTPSender
will only be permitted to use encodings consistent with the offer.